### PR TITLE
Add direct feedback capability

### PR DESCRIPTION
**Pull request by DaveyJonesBitPail**
### CZI Shrink changes
Add a set of buttons that will open a browser to: 
- GitHub repository -> [https://github.com/ZEISS/czicompress/tree/main/czishrink](https://github.com/ZEISS/czicompress/tree/main/czishrink)
- Report an issue -> [https://github.com/ZEISS/czicompress/issues/new/choose](https://github.com/ZEISS/czicompress/issues/new/choose)
- Feedback/Questions -> [https://github.com/ZEISS/czicompress/discussions/new/choose](https://github.com/ZEISS/czicompress/discussions/new/choose)

devenv_l5SGmrvqaf

### Discussion template changes
 - Added discussion template for the following topics for both CZI Shrink and CZI Compress:
   - Feedback
   - Feature Request
   - Questions

### Issue template issue
- Added a config.yml to add "external" links in order to link to relevant discussion dialogs in case users wish to give feedback, ask questions, or add a feature request
- Changed the bug report template from markdown to yaml
  - This will add specific input fields/types which can be made required
chrome_CpR0dTXiPQ

### Auto-label workflow
- Added new workflow to auto-label new bug reports based on tool affected option selected in new template. This can also be tested in temporary test repo: [https://github.com/DaveyJonesBitPail/SampleEmptyRepo](https://github.com/DaveyJonesBitPail/SampleEmptyRepo). Just create a new issue.

TODOs:
- [x] Add Unit Test if applicable of OpenUrl command that use FileChooser
- [x] Update Issue templates for users not familiar with github issue procedure

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## If you want to preview template changes I have temporarily created a repository to enable this before merge: [https://github.com/DaveyJonesBitPail/SampleEmptyRepo](https://github.com/DaveyJonesBitPail/SampleEmptyRepo).

Specifically take a look at: 
 - [https://github.com/DaveyJonesBitPail/SampleEmptyRepo/issues/new/choose](https://github.com/DaveyJonesBitPail/SampleEmptyRepo/issues/new/choose)
 - [https://github.com/DaveyJonesBitPail/SampleEmptyRepo/discussions/categories](https://github.com/DaveyJonesBitPail/SampleEmptyRepo/discussions/categories)
 
 Note that the order of discussion elements may not be the same

## How Has This Been Tested?

Tested locally on Win10 system by using new button.

## Checklist:

- [x] I followed the Contributing Guidelines.
- [ ] I did a self-review.
- [ ] I commented my code, particularly in hard-to-understand areas.
- [ ] I updated the documentation.
- [ ] I updated the version of czicompress following [the README (Versioning)](../README.md#versioning) depending on the [type of change](#type-of-change)
  - Bug fix -> PATCH
  - New feature -> MINOR
  - Breaking change -> MAJOR
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Review by m-ringler — DISMISSED** [review]

File: .github/DISCUSSION_TEMPLATE/czi-compress-questions.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+labels: ["czicompress", "question"]
+body:
+  - type: textarea
+    id: question
+    attributes:
+      label: Question
+      description: Please add any screenshots of logs as applicable
```
Comment (m-aXimilian) [inline .github/DISCUSSION_TEMPLATE/czi-compress-questions.yml:8]: OR  logs?

**Review by m-aXimilian — DISMISSED** [review]
Read the texts and had a look at your sample repository. Looks good to me :)

File: .github/DISCUSSION_TEMPLATE/czi-compress-questions.yml (hunk 1 above)
Comment (DaveyJonesBitPail) [reply .github/DISCUSSION_TEMPLATE/czi-compress-questions.yml]: Added in [57dc64d](https://github.com/ZEISS/czicompress/pull/45/commits/57dc64dd9f006ba1327b42eadba31d0f50d450ab)

**Review by swg08 — APPROVED** [review]
Very good!

**Review by sdierg — DISMISSED** [review]

**Review by ptahmose — APPROVED** [review]
if it works then it LGTM

**Pull request merged**